### ISSUE TITLE
Recording during script execution

### ISSUE THREAD

**Issue by nopnop2002** [rpdswtk/mqtt_recorder#9]
Hello.
I am running the script below.
However, no recording file is created while the script is running.

I would like to see the recording file in real time using tail command.

I want to keep mqtt-recorder running for several months.
```
#!/bin/bash
file=`date '+%Y%m%d_%H%M%S'`
file="/home/nop/recording_"${file}".csv"
echo /home/nop/.local/bin/mqtt-recorder --host 192.168.10.40 --mode record --file ${file} --topics /home/nop/topics.json
/home/nop/.local/bin/mqtt-recorder --host 192.168.10.40 --mode record --file ${file} --topics /home/nop/topics.json
```

**Comment by rpdswtk** [comment]
Hi, writing to csv file continuously have been implemented in #13. Thanks @rcastill. This is now included in [version 1.4.0 ](https://pypi.org/project/mqtt-recorder/1.4.0/)

**Comment by nopnop2002** [comment]
Thank you.
